src/FirstPayment/FirstPaymentBuilder.php

```diff
@@ -6,10 +6,14 @@
 use Illuminate\Support\Str;
 use Laravel\Cashier\Cashier;
 use Laravel\Cashier\FirstPayment\Actions\ActionCollection;
+use Laravel\Cashier\Traits\PaymentMethodString;
```

Comment:
This trait is specific to the FirstPayment, so please move it to the `Laravel\Cashier\FirstPayment` namespace.

Reply:
Moved it to `Laravel\Cashier\FirstPayment\Traits`